### ISSUE TITLE
TypeError from logger setup

### ISSUE THREAD

**Issue by ptone** [thombashi/tcconfig#132]
I was seeing if I could run this in a Docker container

`docker run --rm -it python /bin/bash`

`pip install tcconfig` ok

```
# python
Python 3.8.1 (default, Feb  2 2020, 08:37:37) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> 


root@4f3fcad7f2a3:/# tcset --device eth0 --delay 1sec
Traceback (most recent call last):
  File "/usr/local/bin/tcset", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/site-packages/tcconfig/tcset.py", line 329, in main
    initialize_cli(options)
  File "/usr/local/lib/python3.8/site-packages/tcconfig/_common.py", line 92, in initialize_cli
    set_log_level(options.log_level)
  File "/usr/local/lib/python3.8/site-packages/tcconfig/_logger.py", line 63, in set_log_level
    set_logger(is_enable=True)
  File "/usr/local/lib/python3.8/site-packages/tcconfig/_logger.py", line 39, in set_logger
    subprocrunner.set_logger(is_enable)
  File "/usr/local/lib/python3.8/site-packages/subprocrunner/_logger/_logger.py", line 49, in set_logger
    logger.enable(MODULE_NAME)
TypeError: enable() takes 1 positional argument but 2 were given
```

I get the same init error for every cmd

**Comment by ptone** [comment]
and just to clarify - I understand you support wrapping a docker run with tcconfig - but in this case I'm hoping to apply it from in a running container to other processes, need privs of course

**Comment by thombashi** [comment]
@ptone 
Thank you for your report.

The problem fixed at `tcconfig 0.25.0`.
Please try to upgrade.

FYI: You could use `tcconfig` built-in Docker image: https://hub.docker.com/r/thombashi/tcconfig

**Comment by thombashi** [comment]
I'll close the issue.
Feel free to reopen if you still have any problems about the issue.
